ui/App/ProfileScreen.svelte

```diff
@@ -194,75 +265,85 @@
           modal.toggle(SearchModal);
         }} />
     {:else}
-      <ul class="grid" data-cy="project-list">
-        {#each $profileProjectsStore.data.cloned as project}
-          <li>
-            <ProjectCardSquare
-              {project}
-              isMaintainer={isMaintainer(session.identity.urn, project)}
-              on:click={() => openProject({ detail: project })} />
-          </li>
-        {/each}
-        {#each $profileProjectsStore.data.follows as project}
-          <li>
-            <ProjectCardSquare
-              isMaintainer={isMaintainer(session.identity.urn, project)}
-              {project}
-              on:click={() => openProject({ detail: project })} />
-          </li>
-        {/each}
-        {#if $profileProjectsStore.data.requests.length > 0}
-          <li class="box requests">
-            <div>
-              <h2>Still looking...</h2>
-              <p style="margin-top: 1rem;">
+      <div class="sidebar-layout" class:one-column={!showSidebar}>
```

Comment:
I haven't checked thoroughly, but isn't there a chance we can extract the `sidebar-layout` into the `ScreenLayout` component and provide it via a slot? We already have a similar sidebar in the org screen, this might be a possibility to abstract it out.